packages/@o3r/rules-engine/schemas/rulesets.schema.json

```diff
@@ -166,11 +166,31 @@
             },
             {
               "$ref": "https://raw.githubusercontent.com/AmadeusITGroup/otter/main/packages/%40o3r/components/schemas/rules-engine.placeholder-action.schema.json"
+            },
+            {
+              "$ref": "#/definitions/RulesEngineCustomActionBlock"
             }
           ]
         }
       ]
     },
+    "RulesEngineCustomActionBlock": {
+      "type": "object",
+      "description": "Content of a custom action",
+      "properties": {
+        "actionType": {
+          "type": "string",
+          "description": "Action Type",
+          "pattern": "^(?!(?:SET_FACT|UPDATE_PLACEHOLDER|UPDATE_CONFIG|UPDATE_ASSET|UPDATE_LOCALISATION)$).*$"
+        },
+        "value": {}
+      },
+      "additionalItems": true,
+      "required": [
+        "actionType",
+        "value"
```

Comment:
Not sure it should be required

Reply:
I think it has to be specified but with any value 
![image](https://github.com/user-attachments/assets/3667a7b9-b416-4ba5-a780-f37552611b77)
![image](https://github.com/user-attachments/assets/7070b046-3449-471d-8770-1550229dca4d)